cellpack/autopack/utils.py

```diff
@@ -1,5 +1,139 @@
+import collections
 import numpy
 
 
 def get_distance(pt1, pt2):
     return numpy.linalg.norm(pt2 - pt1)
+
+
+def ingredient_compare1(x, y):
+    """
```

Comment:
Can we rename these functions to reflect the actual difference? Looks like priority > 0 vs priority < 0

Reply:
Or perhaps combine all three into a single function that handles all cases?

Reply:
Pass in another argument that determines which regime we use to break ties based on priority?